### PR TITLE
Design merge

### PR DESCRIPTION
**Pull request by brbrakus**

### CODE REVIEW

File: styles/aboutUs.scss [hunk 1]
```diff
@@ -3,81 +3,123 @@
   width: 100%;
   width: 100vw;
 
-  padding-top: 100px;
-  margin-bottom: 100px;
+  padding-top: $spc-l;
```
Comment (MiroDojkic) [inline styles/aboutUs.scss]: Nice. When you have multiple paddings or margins for the same element shorthand is handy and better to use, for example:
`padding: $spc-l 0 $spc-xl 0;`

File: styles/aboutUs.scss [hunk 2]
```diff
@@ -3,81 +3,123 @@
...
-  background-repeat: no-repeat;
+  color: $font-color;
 
   .about-us-header {
     display: flex;
-    width: 100%;
-    width: 100vw;
-    margin-bottom: 60px;
-    flex-flow: column nowrap;
+    width: 75px;
```
Comment (MiroDojkic) [inline styles/aboutUs.scss]: You can replace this with `rem`

File: styles/aboutUs.scss [hunk 3]
```diff
@@ -3,81 +3,123 @@
...
+    width: 75vw;
+
+    margin-bottom: 100px;
```
Comment (MiroDojkic) [inline styles/aboutUs.scss]: This as well

File: styles/aboutUs.scss [hunk 4]
```diff
@@ -3,81 +3,123 @@
...
     .about-us-card {
-      width: 34%;
-      width: 24vw;
-      max-width: 344px;
-      margin: 40px;
+      width: 75%;
+      width: 75vw;
+      max-width: 550px;
+
+      margin: 0px;
```
Comment (MiroDojkic) [inline styles/aboutUs.scss]: You can just set it to `0`, no need for `px`

File: styles/aboutUs.scss [hunk 5]
```diff
@@ -3,81 +3,123 @@
...
+
+          justify-content: center;
+          align-items: center;
+
+          text-align: center;
+          font-size: 2.125rem;
+          color: $link-hovered-color;
+
+          border-radius: 100%;
+          box-shadow: 3px 3px 6px $text-shadow-color;
```
Comment (MiroDojkic) [inline styles/aboutUs.scss]: This can be refactored to a variable, it's used in multiple places

File: styles/footer.scss [hunk 6]
```diff
@@ -2,76 +2,43 @@ footer {
   display: flex;
   width: 100%;
   width: 100vw;
-  height: 65px;
 
-  flex-direction: row;
-  justify-content: space-between;
+  height: 150px;
```
Comment (MiroDojkic) [inline styles/footer.scss]: `rem`

**Comment by MiroDojkic** [comment]
You should break this down into multiple pull requests instead of single big one, otherwise 🥇 

Please compress the members image and fix minor stuff before merge.

File: styles/aboutUs.scss (hunk 2 above)
Comment (brbrakus) [reply styles/aboutUs.scss]: As you can see in line under, I used 75vw for the same property. I will replace px with % in that example.

File: styles/aboutUs.scss (hunk 3 above)
Comment (brbrakus) [reply styles/aboutUs.scss]: Maybe better solution is to replace it with variable $spc-l, and then replace px with rem in variable value.

File: styles/footer.scss (hunk 6 above)
Comment (brbrakus) [reply styles/footer.scss]: $spc-xl 💃

File: styles/aboutUs.scss (hunk 2 above)
Comment (MiroDojkic) [reply styles/aboutUs.scss]: :see-no-evil:

Comment (MiroDojkic) [reply styles/aboutUs.scss]: 🙈

File: styles/aboutUs.scss (hunk 3 above)
Comment (MiroDojkic) [reply styles/aboutUs.scss]: Sure, that sounds good 👍

File: styles/equipmentSection.scss [hunk 7]
```diff
@@ -2,53 +2,55 @@
...
       background-position-y: 80px;
       background-repeat: no-repeat;
-      background-color: $color-white;
+      background-color: $white;
 
-      color: $link-color;
+      color: $font-color;
       border-radius: 5px;
-      box-shadow: inset 0px 0px 20px 20px $text-shadow-color;
+      box-shadow: 0 7px 14px $card-shadow-color, 0 4px 7px $card-shadow-color;
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss:48]: Good change to create a mixin

**Review by MiroDojkic — APPROVED** [review]

**Pull request merged**